conans/test/integration/command/info/info_test.py

```diff
@@ -55,6 +55,49 @@ class MyTest(ConanFile):
               package_id: {pref.package_id}
               prev: {pref.revision}""") in client.out
 
+    def test_nontuple_topics(self):
+        client = TestClient()
+        # This is the representation that should always happen,
+        # we wouldn't expect topics not to be a tuple here
+        conanfile = textwrap.dedent("""
+                            from conan import ConanFile
+
+                            class MyTest(ConanFile):
+                                name = "pkg"
+                                version = "0.2"
+                                provides = ("bar",)
+                                topics = ("foo",)
+                            """)
+        client.save({"conanfile.py": conanfile})
+        client.run("graph info . --format=json")
+        recipe = json.loads(client.stdout)["nodes"][0]
+        assert type(recipe["topics"]) == list
```

Comment:
type checks are redudant if the line below you are explicitly comparing against a list

Reply:
Yes, I know. The assert is there just in case it ever fails, the diff error message is clearer